src/emulator/downloadableEmulators.spec.ts

```diff
@@ -35,6 +38,8 @@ describe("downloadDetails", () => {
     process.env["FIRESTORE_EMULATOR_BINARY_PATH"] = tempEnvVars["firestore"];
     process.env["DATABASE_EMULATOR_BINARY_PATH"] = tempEnvVars["database"];
     process.env["PUBSUB_EMULATOR_BINARY_PATH"] = tempEnvVars["pubsub"];
+    Object.defineProperty(process, "platform", { value: originalProcessPlatform });
```

Comment:
Please use sinon to mock fields. Don't do it manually. See the other tests in the repo